pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.socialguide</groupId>
     <artifactId>hive-udfs</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>2.0.0</version>
```

Comment:
Why that big difference in version number?

Reply:
because there's a big difference in the required Hive version.